src/SFA.DAS.ApprenticeCommitments/Data/Models/Revision.cs

```diff
@@ -61,6 +61,7 @@ public void Confirm(Confirmations confirmations, DateTimeOffset time)
                     && HowApprenticeshipDeliveredCorrect == true)
                 {
                     ConfirmRevision(time);
+                    Apprenticeship.ConfirmedOn = time.DateTime;
```

Comment:
Just curious, as to whether updating another entity (Aprenticeship_ with the Revisions entity is considered good practice.